env/backend.env

```
@@ -8,4 +8,5 @@ WAIT_HOSTS=mysql_db:3306
 WAIT_HOSTS_TIMEOUT=300
 WAIT_SLEEP_INTERVAL=5
 WAIT_HOST_CONNECT_TIMEOUT=30
-LOG_LEVEL=INFO
+JWT_PUB_KEY=NL5U0AbuR5RE7mDRWFWMUh5ArPa0WGww
```

```suggestion
JWT_PUB_KEY=supersecretkey
```
To avoid conflicts when starting from docker and starting outside a container

---

The docker does not use the env variable?

---

Yes and it would be easier if it was the same as the default value in the backend. That way if we either start from docker or not the password will be the same